lib/jobs/ansible-job.js

```diff
@@ -3,7 +3,10 @@
 'use strict';
 
 module.exports = ansibleJobFactory;
-var di = require('di');
+var di = require('di'),
+    os = require('os'),
+    fs = require('fs'),
```

Comment:
The `fs` has been included in the injectable table, you can move it to below list `di.inject(...)`